Gopkg.lock

```diff
@@ -513,6 +513,14 @@
   revision = "636bf0302bc95575d69441b25a2603156ffdddf1"
   version = "v1.1.1"
 
+[[projects]]
+  digest = "1:68c64bb61d55dcd17c82ca0b871ddddb5ae18b30cfe26f6bfd4b6df6287dc2e0"
+  name = "github.com/golang/mock"
```

Comment:
Looks like this is unused, is this still in the plans to use?

Reply:
It's a new dependency of gosnmp.  I updated to latest gosnmp to get a recent change related to printing the type of a trap variable.